azure-pipelines.bundlesize.yml

```diff
@@ -25,17 +25,28 @@ jobs:
         displayName: yarn
 
       - script: |
-          yarn build --no-cache $(sinceArg)
-        displayName: build packages
+          yarn lage bundle-size --no-cache --verbose $(sinceArg)
```

Comment:
We should run `bundle-size` command while `build` is only a dependency (I also modified `lage.config.js`) otherwise it could cause build failures due scoped builds.

For example:
- `yarn build --no-cache` will run build for `react-menu` only
- `yarn lage bundle-size --no-cache` will run `bundle-size` for all packages (`react-menu`, `react-image`, etc.)
- As required artifacts are not present (for `react-image`), build will fail

Reply:
Yeah, you're right 🤔 As `build` is present on _every_ package it should not a be thing, scopes will be also same.

So current change will give us only [better concurrency](https://microsoft.github.io/lage/guide/levels.html#level-4-pipelining) as `lage` will run `bundle-size` command immediately once all required `build`s are completed. Should I revert it?